src/Uno.UI.Tests/Windows_UI_Xaml/Given_StaticResource.cs

```diff
@@ -19,9 +25,452 @@ public void When_Resource_In_Application_Merged()
+			Assert.AreEqual(256, noDP);
+		}
+
+		[TestMethod]
+		public void When_Non_View_And_Local_Brush()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var control = new Test_Control();
+
+			Assert.AreEqual(Colors.MintCream, (control.TemplateFromResourceControl.Foreground as SolidColorBrush).Color);
+		}
+
+		[TestMethod]
+		public void When_Non_View_And_Local_Resource()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var control = new Test_Control();
+			app.HostView.Children.Add(control);
+
+			var rowDef = control.TestGrid.RowDefinitions[1];
+			Assert.AreEqual(333, rowDef.Height.Value);
+		}
+
+		[TestMethod]
+		public void When_ResourceKey_And_System()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var res = app.Resources["ImAStaticResourceInADictColorBrush"] as SolidColorBrush;
+			Assert.AreEqual(Colors.LightGray, res.Color);
+		}
+
+		[TestMethod]
+		public void When_ResourceKey_And_Local()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var res = app.Resources["ResourceKeyLocalReference"] as SolidColorBrush;
+			Assert.AreEqual(Colors.MediumSpringGreen, res.Color);
+		}
+
+		[TestMethod]
+		public void When_ResourceKey_And_Assigned()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var template = app.Resources["TemplateUsingStaticResourceAlias"] as ControlTemplate;
+			var button = new Button { Template = template };
+			button.Measure(new Size(1000, 1000));
+			var tb = button.FindFirstChild<TextBlock>();
+			Assert.AreEqual(Colors.MediumSpringGreen, (tb.Foreground as SolidColorBrush).Color);
+		}
+
+		[TestMethod]
+		public void When_Converter()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var page = new Test_Page();
+
+			page.DataContext = new
+			{
+				Boolean1 = true,
+				Boolean2 = false
+			};
+
+			AssertEx.AssertHasColor(page.TestBorder.Background, Colors.Plum);
+		}
+
+		[TestMethod]
+		public void When_Converter_In_Template()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var page = new Test_Page();
+			page.DataContext = new
+			{
+				Boolean1 = true,
+				Boolean2 = false
+			};
+
+			app.HostView.Children.Add(page);
+			page.Measure(new Size(1000, 1000));
+
+			var tb = page.TestContentControl.FindFirstChild<TextBlock>();
+			Assert.AreEqual("Inner text", tb.Text);
+			AssertEx.AssertHasColor(tb.Foreground, Colors.Tomato);
+		}
+
+		[TestMethod]
+		public void When_Converter_In_Template_Separate_Xaml()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+
+			var page = new Test_Page();
+
+			page.SpiffyItemsControl.ItemsSource = Enumerable.Range(0, 3).Select(i => true).ToArray();
+
+			app.HostView.Children.Add(page);
+			page.Measure(new Size(1000, 1000));
+
+			var rb = page.SpiffyItemsControl.FindFirstChild<RadioButton>();
+			AssertEx.AssertHasColor(rb.Foreground, Colors.Plum);
+			;
+		}
+
+		[TestMethod]
+		public void When_Implicit_Conversion()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'app'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)